regression/goto-analyzer/constant_propagation_19/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
```

Comment:
Would it be possible to call this new test something a little more descriptive, such as `constant_propagation_two_way_1` or something?

Reply:
Done.